mozilla-release/browser/components/sessionstore/SessionStore.jsm

```diff
@@ -2094,8 +2094,9 @@ var SessionStoreInternal = {
     // Create a new tab.
     let userContextId = aTab.getAttribute("usercontextid");
     let newTab = aTab == aWindow.gBrowser.selectedTab ?
-      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab, userContextId}) :
-      aWindow.gBrowser.addTab(null, {userContextId});
+      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab,
+                                      userContextId, private: aTab.private}) :
```

Comment:
Weird formatting. Please see https://github.com/airbnb/javascript#whitespace for examples